filebeat/prospector/redis/doc.go

```diff
@@ -0,0 +1,14 @@
+// Redis package contains prospector and harvester to read the redis slow log
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
package comment should be of the form "Package redis ..."